### PR TITLE
Enable preview features for .net6 ITs

### PR DESCRIPTION
**Pull request by csaba-sagi-sonarsource**

### CODE REVIEW

File: analyzers/its/sources/Net6/Net6/Net6.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>disable</Nullable>
-    <LangVersion>preview</LangVersion>
```
Comment (costin-zaharia-sonarsource) [inline analyzers/its/sources/Net6/Net6/Net6.csproj:8]: It was intended to remove this?

Comment (csaba-sagi-sonarsource) [reply analyzers/its/sources/Net6/Net6/Net6.csproj:8]: Yes. In VS 2022-preview with that line the preview features were enabled at the build was successful, however it changed and the build was failing. Now there is this new way to enable the preview features.

Comment (costin-zaharia-sonarsource) [reply analyzers/its/sources/Net6/Net6/Net6.csproj:8]: As far as I can tell these are 2 very different properties, `LangVersion` is related to the language version, and `EnablePreviewFeatures` is related to APIs. See: https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#enabling-a-preview-feature

Comment (costin-zaharia-sonarsource) [reply analyzers/its/sources/Net6/Net6/Net6.csproj:8]: In any case, we don't really need `<LangVersion>preview</LangVersion>` now so I can merge it like it is. I was though curious if there was a reason behind the change.

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**